### PR TITLE
feat: Enable startup logging to EventLog for .NET 6+ applications on Windows.

### LINKED ISSUE #1939: Enable startup logging to EventLog on .NET 6+ under Windows
### Describe the story, please be clear on scope of the story.
Agent startup logging currently only writes to the Windows EventLog when running under .NET Framework. We should enable writing to EventLog when running under .NET 6+ on Windows as well. 

### Acceptance Criteria
Startup logging writes to Windows EventLog for .NET 6+ apps running on windows.

### Estimates
Please provide initial t-shirt size
S

### PR DESCRIPTION
**Pull request by tippmar-nr**
Enables writing to Windows EventLog on .NET 6+ applications during startup. 

Also enables fallback to EventLog in case the file log can't be created.

Resolves #1939

### CODE REVIEW

File: src/Agent/NewRelic/Agent/Core/Logging/LoggerBootstrapper.cs [hunk 1]
```diff
@@ -107,27 +108,38 @@ private static LoggerConfiguration ConfigureInMemoryLogSink(this LoggerConfigura
...
-        /// Add the Event Log sink if running on .NET Framework
+        /// Add the Event Log sink if running on Windows
         /// </summary>
         /// <param name="loggerConfiguration"></param>
         private static LoggerConfiguration ConfigureEventLogSink(this LoggerConfiguration loggerConfiguration)
         {
-#if NETFRAMEWORK
-            const string eventLogName = "Application";
-            const string eventLogSourceName = "New Relic .NET Agent";
+#if NETSTANDARD2_0
```
Comment (chynesNR) [inline src/Agent/NewRelic/Agent/Core/Logging/LoggerBootstrapper.cs:116]: We also want this on Framework, right?

Comment (tippmar-nr) [reply src/Agent/NewRelic/Agent/Core/Logging/LoggerBootstrapper.cs:116]: Oh yeah, I guess I just need to #ifdef around the OS checking. Good catch.

File: src/Agent/NewRelic/Agent/Core/Logging/LoggerBootstrapper.cs [hunk 2]
```diff
@@ -107,27 +108,38 @@ private static LoggerConfiguration ConfigureInMemoryLogSink(this LoggerConfigura
...
+                                    logName: eventLogName,
+                                    restrictedToMinimumLevel: LogEventLevel.Warning,
+                                    outputTemplate: "{Level}: {Message}{NewLine}{Exception}",
+                                    manageEventSource: true
+                                );
+                        });
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e);
```
Comment (chynesNR) [inline src/Agent/NewRelic/Agent/Core/Logging/LoggerBootstrapper.cs:140]: I'm not sure automatically writing to the console is a good idea, even in an emergency, but I'm curious what others think.

**Comment by tippmar-nr** [comment]
Closed after it was pointed out that the solution won't work unless the app is running as administrator. Grr.

**Pull request closed without merging**